contracts/deploy/03-routers/03-arb-to-gnosis-router.ts

```diff
@@ -39,11 +39,30 @@ const deployRouter: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const hardhatDeployer = async () => {
     const veaOutbox = await deployments.get("VeaOutboxArbToGnosis");
     const veaInbox = await deployments.get("VeaInboxArbToGnosis");
+    const amb = await deployments.get("MockAMB");
+    //const ArbSysMock = await deployments.get('arbSysMock');
```

Comment:
Just checking: is this commented line intentional or a leftover that slipped?

Reply:
reminisce of what was before

Reply:
IMHO, we would be doing fine by removing code that does not get executed (commented, unreachable, etc). With git we can always go back in history if we decide it was crucial, and it helps to keep the codebase tidy